src/expression-modifiers/helper.js

```diff
@@ -370,4 +394,18 @@ export default {
   getExpressionWithMarkers,
 
   canExtract,
+
```

Comment:
We should avoid adding stuff in here that is not being used from multiple places (no purpose to put it in here then). Instead, place specific functions closer to where they are used.

Reply:
Moved functions to place where use them